doc/source/whatsnew/v2.2.0.rst

```diff
@@ -45,8 +45,8 @@ enhancement2
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
+- :meth:`pandas.tseries.api.guess_datetime_format` is now available (:issue:`54727`)
```

Comment:
```suggestion
- :meth:`pandas.tseries.guess_datetime_format` is now available (:issue:`54727`)
```

Reply:
Is this because guess_datetime_format is now a public method - and no longer needs to be called through `api`?

Reply:
Oh sorry, I'm actually incorrect. I see you added it to `api`.